src/installer/pkg/packaging/snaps/dotnet-runtime-3.1/snap/snapcraft.yaml

```diff
@@ -0,0 +1,45 @@
+name: dotnet-runtime-31
+base: core18
+version: 3.1.0
+summary: Cross-Platform .NET Core Runtime. 
+description: |
+  .NET Core runtimes and libraries which are optimized for running .NET Core apps in production. See https://dot.net/core.
+  .NET Core is a general purpose development platform maintained by Microsoft. 
+
+grade: stable
+confinement: strict
+
+apps:
+  dotnet:
+    command: dotnet
+    plugs:
+      - network
+      - network-bind
+      - removable-media
+      - home
+
+slots:
+  dotnet-runtime:
+    content: dotnet-runtime-31
+    interface: content
+    read: [/]
+
+parts:
+  dotnet-runtime:
+    plugin: dump
+    source: https://download.visualstudio.microsoft.com/download/pr/30915c37-fa5a-4930-b4e6-b4130e4596b2/38d531c10dc56950f17f3c604e9a2ebc/aspnetcore-runtime-3.1.0-linux-x64.tar.gz
+    source-checksum: sha512/7d14e4617413fd7939f90a4e283b7abedbd51ecfbc150938d4c52ecca8182cab8c4946408c1f6369944c68757c91b580796d9e0c62349c377c5e90b739a87d8b
+    stage-packages:
+      - libicu60
+      - libssl1.0.0
+      - libcurl3
+      - libgssapi-krb5-2
+      - liblttng-ust0
+      - libstdc++6
+      - zlib1g
+      - libgcc1
+      - lldb
+      - libunwind8
```

Comment:
I think libunwind8 and few other dependencies are not required for 3.0 onwards? cc @janvorli

Reply:
Correct, libunwind8 need was dropped, actually in 2.1 already.
The lldb isn't needed from 3.0 on, since the libsosplugin is not part of the runtime / sdk anymore.
As for the libssl, we support running with libssl1.0.0, 1.1.0 and 1.1.1. On some distros, the 1.0.0 is not available anymore and only 1.1.0/1.1.1 are present. I am not familiar with how snap works though, so I am not sure if that's a problem or not.

Reply:
This would also apply to the SDK image.

Reply:
I can make the suggested changes, but do we have approval to merge this in runtime repo?

Reply:
Fixed